ci/x86_64-windows-debug.ps1

```diff
@@ -1,13 +1,21 @@
-$TARGET = "$($Env:ARCH)-windows-gnu"
+$ARCHITECTURE = switch ([Environment]::Is64BitOperatingSystem) {
+    $true { "x86_64"; break }
+    $false { "x86"; break }
+}
+
+$TARGET = "$ARCHITECTURE-windows-gnu"
 $ZIG_LLVM_CLANG_LLD_NAME = "zig+llvm+lld+clang-$TARGET-0.11.0-dev.1869+df4cfc2ec"
 $MCPU = "baseline"
 $ZIG_LLVM_CLANG_LLD_URL = "https://ziglang.org/deps/$ZIG_LLVM_CLANG_LLD_NAME.zip"
 $PREFIX_PATH = "$(Get-Location)\$ZIG_LLVM_CLANG_LLD_NAME"
 $ZIG = "$PREFIX_PATH\bin\zig.exe"
 $ZIG_LIB_DIR = "$(Get-Location)\lib"
 
-Write-Output "Downloading $ZIG_LLVM_CLANG_LLD_URL"
-Invoke-WebRequest -Uri "$ZIG_LLVM_CLANG_LLD_URL" -OutFile "$ZIG_LLVM_CLANG_LLD_NAME.zip"
+if (-not(Test-Path -Path "$ZIG_LLVM_CLANG_LLD_NAME.zip" -PathType Leaf)) {
+    Write-Output "Downloading $ZIG_LLVM_CLANG_LLD_URL"
+    Invoke-WebRequest -Uri "$ZIG_LLVM_CLANG_LLD_URL" -OutFile "$ZIG_LLVM_CLANG_LLD_NAME.zip"
+
+}
```

Comment:
This is not helpful because this script unfortunately must download the tarball with every run. It is run on GitHub Actions, not our own hardware.

Reply:
Makes sense. I started using the script when started looking at zig yesterday, because it was very convenient. I guess updating the document a bit will make it easier for anyone starting out on windows.